drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -336,11 +336,10 @@ def self.find_definition_for_service_type(service_type, project:)
     Hmis::Form::Definition.for_project(project: project, role: :SERVICE, service_type: service_type)
   end
 
-  # Validate JSON definition when loading, to ensure no duplicate link IDs
-  def self.validate_json(...)
-    Hmis::Form::DefinitionValidator.perform(...).each do |issue|
-      yield issue
-    end
+  # returns list of HmisErrors::Error objects
+  # rename to like validate_form or something?
+  def validate_json
+    Hmis::Form::DefinitionValidator.perform(definition, role)
   end
```

Comment:
+gig update comment to be more descriptive.
This is the main validation method for validating the form content, including it's JSON structure, HUD conformance (if applicable), etc.